### DISCUSSION TITLE
Keyboard dependent specific config?

### DISCUSSION THREAD

**Discussion by alxsimon** [helix-editor/helix#6244] [Q&A]
Is there a way to have a different config (remappings etc.) based on which keyboard is used in the system.

I have an external keyboard set to Colemak-DH but also use the laptop keyboard when not at my desk.
It would be great if I could keep the same helix "layout" between the two, though I'm not sure how easy it would be to remember everything.

**Comment by cd-a** [comment]
I propose setting your keyboard layout on the laptop to ColemakDH as well. I've done the same and it works really well. 

That way you don't have to change any bindings around and get more used to the layout instead of switching all the time. It will get quite messy with the keybinding changes.

In case you've programmed it to the firmware on your external keyboard and lacking the layout in the system for your laptop, you can download it here: https://github.com/ColemakMods/mod-dh

**Reply by alxsimon** [reply]
Thanks for the idea, but not sure I'm ready for that yet. But I'll sure switch at some point in the future.

**Comment by SoraTenshi** [comment]
There's currently this PR open that should address this exact issue: https://github.com/helix-editor/helix/pull/5046
